requirements.txt

```diff
@@ -27,8 +27,6 @@ certifi==2021.5.30
     #   -r requirements.in
     #   elasticsearch
     #   requests
-cffi==1.14.5
```

Comment:
I'm not sure why all of the following changes are getting done in this file.

Reply:
Hmm not sure, probably remove them from this PR.